net.dengine.Doomsday.json

```diff
@@ -0,0 +1,56 @@
+{
+    "app-id": "net.dengine.Doomsday",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "doomsday",
+    "default-branch": "2.2",
+    "modules": [
+        {
+            "name": "fluidsynth",
+            "buildsystem": "cmake",
+            "builddir": true,
+            "config-opts": [
+                "-DLIB_SUFFIX=",
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+                "-Denable-aufile=NO",
+                "-Denable-ipv6=NO",
+                "-Denable-network=NO"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/FluidSynth/fluidsynth/archive/v2.0.9.tar.gz",
+                    "sha256": "bfe82ccf1bf00ff5cfc18e2d9d1e5d95c6bd169a76a2dec14898d1ee0e0fac8a"
+                }
+            ]
+        },
+        {
+            "name": "doomsday",
+            "buildsystem": "cmake",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+                "-DDE_PREFIX=/app",
+                "-DDENG_ENABLE_SDK=NO"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/skyjake/Doomsday-Engine",
+                    "tag": "v2.2.0-1",
+                    "commit": "0acb2c16be15f5cddc713ae6ec37fd28514d5fd1"
+                }
+            ]
+        }
+    ],
+    "finish-args": [
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home",
+        "--filesystem=/usr/share/games:ro",
+        "--filesystem=/etc/timidity:ro"
```

Comment:
Why you need those filesystem access?

Reply:
I will remove these. The first one was supposed to allow automatically finding WAD files under /usr/share/games/doom. The second one presumably is required by SDL_mixer so it can play MIDI music via timidity+freepats. However, further testing shows that neither of these actually works as expected.